helloworld-in-cs/client.py

```diff
@@ -24,11 +26,12 @@ def hello_world(args):
     }
 
     # download the data
-    download_data(targets)
+    download_data(targets, args.download_proxy)
+
 
     # run it!
-    py_client(port_expose=args.port_expose, host=args.host).index(
-        input_fn(targets['index']['filename']), batch_size=args.index_batch_size)
+    py_client(port_expose=args.port_expose, host=args.host, batch_size=args.index_batch_size).index(
+        input_numpy(targets['index']['data']), batch_size=args.index_batch_size)
```

Comment:
targets['index''] does not have `data`

Reply:
Yes, I also get very confused at this. When I print `targets['index']['data']` I also get a key error. But `targets['index']['data']` is how it works, and `targets['index']['filename']`

Reply:
This change to 'data' is inspired by our Jina repo's helloworld code.

Reply:
targets['index']['data'] is add in the `download_data(targets, args.download_proxy)` function in the previous step